tests/zkevm/test_worst_compute.py

```diff
@@ -170,3 +176,115 @@ def test_worst_modexp(
         post={},
         blocks=[Block(txs=[tx])],
     )
+
+
+@pytest.mark.zkevm
```

Comment:
This is no longer needed as it the `zkevm` marker automatically gets applied to all tests in `./tests/zkevm` via the local pytest plugin in `./tests/zkevm/conftest.py` 
```suggestion
```